geopandas/io/sql.py

```diff
@@ -2,6 +2,7 @@
 from contextlib import contextmanager
 
 import pandas as pd
+import sqlalchemy
```

Comment:
`sqlalchemy` is an optional dependency, so it needs to be imported inside the function, like we import parquet here https://github.com/geopandas/geopandas/blob/e62b74a81428e6ab8ced23d688424a83bdabb686/geopandas/io/arrow.py#L311-L313

Reply:
I will fix that!